doc/users-guide/users-guide-crypto.adoc

```diff
@@ -61,6 +90,70 @@ session’s completion queue, which can be accessed directly or via the ODP
 scheduler. The completion event contains the status of the operation and the
 result. The application has the responsibility to free the completion event.
 
+Upon receipt of an `ODP_EVENT_CRYPTO_COMPL` event, the
+`odp_crypto_compl_result()` API is used to retrieve the
+`odp_crypto_op_result_t` associated with the event. This result struct in turn
+contains:
+
+* An indication of the success or failure of the crypto operation
+* The user context associated with the event
+* The output `odp_packet_t`.
+* The `odp_crypto_op_status_t` for the requested cipher operation
+* The `odp_crypto_op_status_t` for the requested authentication operation
+
+==== Packet-based Crypto Operations
+To simplify the original cryptographic operation request API, as well as to
+be more flexible and consistent with the protocol-aware APIs introduced for
+IPsec support, a newer packet-oriented set of cryptographic operation
+APIs is also provided. Applications may use either API set, but going forward
+it is expected that these newer APIs will be the focus of continued
+development.
+
+Instead of a single `odp_crypto_operation()` API, the packet-based APIs
+provide two APIs. `odp_crypto_op()` is the synchronous form while
+`odp_crypto_op_enq()` is the asynchronous form.
```

Comment:
I'd suggest to add the following text here:
To check if ODP implementation supports synchronous or asynchronous crypto packet API, application can check corresponding flags in capabilities structure returned by `odp_crypto_capability()` function.

Otherwise this looks good, thank you!